src/user/user.service.ts

```diff
@@ -38,6 +43,36 @@ export class UserService {
     return this.userRepository.save(user);
   }
 
+  async login(loginDto: LoginDto) {
+    const { password, email } = loginDto;
+    // Verify email
+    const userFoundByEmail = await this.userRepository.findOne({ where: { email } });
+
+    if (!userFoundByEmail) {
+      // if the email doesn't exists, throw a bad request exception
+      throw new HttpException("The user doesn't exists", HttpStatus.BAD_REQUEST);
```

Comment:
You can use [Nest.js builtin exception](https://docs.nestjs.com/exception-filters#built-in-http-exceptions)